src/test/java/uk/gov/ons/ctp/response/action/endpoint/ActionRuleEndpointUnitTest.java

```diff
@@ -206,7 +216,11 @@ public void createActionRuleActionPlanNotFound() throws Exception {
 
     final ResultActions resultActions = mockMvc.perform(postJson("/actionrules", ACTION_RULE_CREATE_VALID_JSON));
 
-    resultActions.andExpect(status().isNotFound());
+    resultActions.andExpect(status().isNotFound())
+            .andExpect(handler().handlerType(ActionRuleEndpoint.class))
+            .andExpect(handler().methodName("createActionRule"))
+            .andExpect(jsonPath("$.error.code", is(CTPException.Fault.RESOURCE_NOT_FOUND.name())))
```

Comment:
:art: `$.error.code` and `$.error.message` are used multiple times in this test class and it may be worth making them a constant